source/agora/flash/Channel.d

```diff
@@ -110,7 +110,7 @@ public class Channel
 
     /// @WORKAROUND@: LocalRest issue with timings and sleep not working until
     /// node ctor is done (see Flash restart tests)
```

Comment:
This has been fixed right ?

Reply:
Yep, we fixed that.
But while this works around that, I think it also fixes the race between channel ctor and peer registering itself to the registry. So we can just remove the workaround comments.